File: Java/Komponen.java
public class Komponen
{
    private String nama;
    private String merk;

    public Komponen()
    {
        this.nama = "";
        this.merk = "";
    }

    public Komponen(String nama, String merk)
    {
        this.nama = nama;
        this.merk = merk;
    }

    public void set_nama(String nama)
    {
        this.nama = nama;
    }

    public void set_merk(String merk)
    {
        this.merk = merk;
    }

    public String get_nama()
    {
        return this.nama;
    }

    public String get_merk()
    {
        return this.merk;
    }
}

File: Python/Komponen.py
class Komponen:
    __nama = ""
    __merk = ""

    def __init__(self, nama, merk):
        self.__nama = nama
        self.__merk = merk

    def get_nama(self):
        return self.__nama
    
    def get_merk(self):
        return self.__merk
    
    def set_nama(self, nama):
        self.__nama = nama

    def set_merk(self, merk):
        self.__merk = merk